egs/librispeech/asr/simple_v1/train_fast.py

```diff
@@ -0,0 +1,286 @@
+#!/usr/bin/env python3
```

Comment:
What's the difference between `train` and `train_fast`?

Reply:
train_fast is for debugging, with smaller minibatch sizes, etc.  We are still getting this to run.